extensions/wikia/EditPageLayout/EditPageService.class.php

```diff
@@ -80,7 +80,7 @@ public function getPreview($wikitext) {
 		 * We still rely on the parser for other stuff
 		 */
 		if ( $this->mTitle->isCssOrJsPage() ) {
-			$html = '<pre>' . $originalWikitext . '</pre>';
+			$html = '<pre>' . htmlspecialchars($originalWikitext) . '</pre>';
```

Comment:
Nitpicky ;) but should be spacey per coding conventions: `htmlspecialchars( $originalWikitext )`